phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java

```diff
@@ -79,13 +81,28 @@ public void showAvailableApps() {
      */
     public void populate(List<CategoryDTO> categories) {
         Platform.runLater(() -> {
+            Set<String> themedCategories = Sets.newHashSet(
+                    "custom",
```

Comment:
I guess we can expect that the "default" categories are contained in the theme (even if they link to the default icons).

I agree that it would be better to load the themed icon whenever it is available but, as I said, I don't know how this can be done.